source/draw/RasterBgra.ooc

```diff
@@ -127,7 +127,8 @@ RasterBgra: class extends RasterPacked {
 		x, y, imageComponents: Int
 		requiredComponents := 4
 		data := StbImage load(filename, x&, y&, imageComponents&, requiredComponents)
-		This new(ByteBuffer new(data as UInt8*, x * y * requiredComponents), IntVector2D new(x, y))
+		buffer := ByteBuffer new(data as UInt8*, x * y * requiredComponents, true)
+		This new(buffer, IntVector2D new(x, y))
```

Comment:
@marcusnaslund : it crashed on your machine the last time we tried . I moved the buffer creation to separate line on purpose, let's see if this helps. I remember when moving initialization to separate line fixed invalid memory read related to Closure ...